app/src/main/java/org/simple/clinic/patient/recent/RecentPatientItem.kt

```diff
@@ -6,12 +6,14 @@ import com.xwray.groupie.ViewHolder
 import kotlinx.android.synthetic.main.recent_patient_item_view.view.*
 import org.simple.clinic.R
 import org.simple.clinic.patient.Gender
+import org.simple.clinic.summary.RelativeTimestamp
 
 data class RecentPatientItem(
     val name: String,
     val age: Int,
     val lastBp: String,
-    val gender: Gender
+    val gender: Gender,
+    val lastBpRelativeTimestamp: RelativeTimestamp?
```

Comment:
This looks wrong. `lastBp` is not `nullable` whereas `lastBpRelativeTimestamp` is.
This can never happen.
Can you instead pass `LastBp` to `RecentPatientItem` and if its `null, make the entire `lastBprecentpatient_item_last_bp.text` as `GONE`?

You can refer `PatientSearchResultsAdapter`

Reply:
I think this is fine because we'll send empty string if `LastBp` happens to be null. And we also get the `RelativeTimestamp` in the controller. We cannot simple send `LastBp` to `RecentPatientItem`. If we do, there'll be some controller logic stuck in `RecentPatientItem`.

Reply:
I've tried this alternate way of creating a new data class `RecentPatientItem.LastBp` to encapsulate all the nullable fields.